.changes/next-release/feature-AWSSDKforJavav2-3a388c7.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We don't need to mention this in the changelog it's an internal detail

Reply:
Removed the changelog